src/runtime_src/core/edge/drm/zocl/zocl_bo.c

```diff
@@ -523,6 +526,11 @@ int zocl_map_bo_ioctl(struct drm_device *dev,
 	args->offset = drm_vma_node_offset_addr(&gem_obj->vma_node);
 	zocl_describe(to_zocl_bo(gem_obj));
 
+	/* Update softkernel memory stats */
+	bo = to_zocl_bo(gem_obj);
+	if (bo->flags & ZOCL_BO_FLAGS_HOST_BO)
+        	zocl_update_sk_mem_stat(zdev, 1, ZOCL_MEM_STAT_TYPE_MAPBO);
```

Comment:
There is always add "1" when calling this function. Do you really need a function that add more than 1?

Maybe use zocl_sk_mem_stat_incr(zdev, ZOCL_MEM_STAT_TYPE_MAPBO); for better readability?

Try avoid constant value and if you have to do so, make the code self readable.

Reply:
Thanks for this suggestion. Updated the code accordingly.